### PR TITLE
[PPP-4865] - Vulnerable Component: snakeyaml

### PR DESCRIPTION
**Pull request by andreramos89**
@smmribeiro 
@renato-s

### CODE REVIEW

**Comment by buildguy** [comment]
<div align='center'>

[👍 Frogbot scanned this pull request and found that it did not add vulnerable dependencies.](https://docs.jfrog-applications.jfrog.io/jfrog-applications/frogbot)

</div>

<details>
<summary> <b>Note:</b> </summary>


---
<div align='center'>

**Frogbot** also supports **Contextual Analysis, Secret Detection, IaC and SAST Vulnerabilities Scanning**. This features are included as part of the [JFrog Advanced Security](https://jfrog.com/advanced-security) package, which isn't enabled on your system.

</div>


</details>


---
<div align='center'>

[🐸 JFrog Frogbot](https://docs.jfrog-applications.jfrog.io/jfrog-applications/frogbot)

</div>

**Comment by buildguy** [comment]
### :warning: Build finished in 20m 31s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -Djs.no.sandbox
```
---
### :no_entry: Failed Tests

#### :cloud_with_lightning_and_rain: 25 test(s) failed:

<details><summary><b>org.pentaho.reporting.ui.datasources.jdbc.ModuleTest.testModuleExists</b> (click to expand)</summary><p>

```
${result.errorDetails}
```

</p></details>

<details><summary><b>org.pentaho.reporting.ui.datasources.kettle.ModuleTest.testEditorRegistered</b> (click to expand)</summary><p>

```
Unable to locate metadata for data-factory type org.pentaho.reporting.engine.classic.extensions.datasources.kettle.KettleDataFactory
```

</p></details>

<details><summary><b>org.pentaho.reporting.ui.datasources.kettle.ModuleTest.testModuleExists</b> (click to expand)</summary><p>

```
${result.errorDetails}
```

</p></details>

<details><summary><b>org.pentaho.reporting.engine.classic.extensions.datasources.kettle.BigDataKettleFactoryTest.testParameter</b> (click to expand)</summary><p>

```
Unable to locate metadata for data-factory type org.pentaho.reporting.engine.classic.extensions.datasources.kettle.KettleDataFactory
```

</p></details>

<details><summary><b>org.pentaho.reporting.engine.classic.extensions.datasources.kettle.BigDataKettleFactoryTest.testQuery</b> (click to expand)</summary><p>

```
Caught Kettle Exception: Check your configuration
```

</p></details>

<details><summary><b>org.pentaho.reporting.engine.classic.extensions.datasources.kettle.BigDataKettleFactoryTest.testSaveAndLoad</b> (click to expand)</summary><p>

```
Unable to parse the document: ResourceKey{schema=org.pentaho.reporting.libraries.docbundle.bundleloader.ZipResourceBundleLoader, identifier=content.xml, factoryParameters={org.pentaho.reporting.libraries.resourceloader.FactoryParameterKey{name=repository-loader}=org.pentaho.reporting.libraries.docbundle.bundleloader.ZipResourceBundleLoader@25e8e59, org.pentaho.reporting.libraries.resourceloader.FactoryParameterKey{name=repository}=org.pentaho.reporting.libraries.repository.zipreader.ZipReadRepository@6bd679b4}, parent=ResourceKey{schema=org.pentaho.reporting.libraries.resourceloader.loader.raw.RawResourceLoader, identifier=[B@1f3b3c33, factoryParameters={}, parent=null}}
```

</p></details>

<details><summary><b>org.pentaho.reporting.engine.classic.extensions.datasources.kettle.BigDataKettleFactoryTest.testSerialize</b> (click to expand)</summary><p>

```
Caught Kettle Exception: Check your configuration
```

</p></details>

<details><summary><b>org.pentaho.reporting.engine.classic.extensions.datasources.kettle.BigDataKettleFactoryTest.testMetaData</b> (click to expand)</summary><p>

```
Unable to locate metadata for data-factory type org.pentaho.reporting.engine.classic.extensions.datasources.kettle.KettleDataFactory
```

</p></details>

<details><summary><b>org.pentaho.reporting.engine.classic.extensions.datasources.kettle.BigDataKettleFactoryTest.testDerive</b> (click to expand)</summary><p>

```
Caught Kettle Exception: Check your configuration
```

</p></details>

<details><summary><b>org.pentaho.reporting.engine.classic.extensions.datasources.kettle.BigDataKettleFactoryTest.testSaveAndLoadForSubReports</b> (click to expand)</summary><p>

```
Unable to parse the document: ResourceKey{schema=org.pentaho.reporting.libraries.docbundle.bundleloader.ZipResourceBundleLoader, identifier=content.xml, factoryParameters={org.pentaho.reporting.libraries.resourceloader.FactoryParameterKey{name=repository-loader}=org.pentaho.reporting.libraries.docbundle.bundleloader.ZipResourceBundleLoader@25e8e59, org.pentaho.reporting.libraries.resourceloader.FactoryParameterKey{name=repository}=org.pentaho.reporting.libraries.repository.zipreader.ZipReadRepository@5d8a0b6}, parent=ResourceKey{schema=org.pentaho.reporting.libraries.resourceloader.loader.raw.RawResourceLoader, identifier=[B@6b8564e5, factoryParameters={}, parent=null}}
```

</p></details>

<details><summary><b>org.pentaho.reporting.engine.classic.extensions.datasources.kettle.KettleDataFactoryTest.testParameter</b> (click to expand)</summary><p>

```
Unable to locate metadata for data-factory type org.pentaho.reporting.engine.classic.extensions.datasources.kettle.KettleDataFactory
```

</p></details>

<details><summary><b>org.pentaho.reporting.engine.classic.extensions.datasources.kettle.KettleDataFactoryTest.testQuery</b> (click to expand)</summary><p>

```
Caught Kettle Exception: Check your configuration
```

</p></details>

<details><summary><b>org.pentaho.reporting.engine.classic.extensions.datasources.kettle.KettleDataFactoryTest.testSaveAndLoad</b> (click to expand)</summary><p>

```
Unable to parse the document: ResourceKey{schema=org.pentaho.reporting.libraries.docbundle.bundleloader.ZipResourceBundleLoader, identifier=content.xml, factoryParameters={org.pentaho.reporting.libraries.resourceloader.FactoryParameterKey{name=repository-loader}=org.pentaho.reporting.libraries.docbundle.bundleloader.ZipResourceBundleLoader@25e8e59, org.pentaho.reporting.libraries.resourceloader.FactoryParameterKey{name=repository}=org.pentaho.reporting.libraries.repository.zipreader.ZipReadRepository@68f1b06b}, parent=ResourceKey{schema=org.pentaho.reporting.libraries.resourceloader.loader.raw.RawResourceLoader, identifier=[B@56c802bc, factoryParameters={}, parent=null}}
```

</p></details>

<details><summary><b>org.pentaho.reporting.engine.classic.extensions.datasources.kettle.KettleDataFactoryTest.testSerialize</b> (click to expand)</summary><p>

```
Caught Kettle Exception: Check your configuration
```

</p></details>

<details><summary><b>org.pentaho.reporting.engine.classic.extensions.datasources.kettle.KettleDataFactoryTest.testMetaData</b> (click to expand)</summary><p>

```
Unable to locate metadata for data-factory type org.pentaho.reporting.engine.classic.extensions.datasources.kettle.KettleDataFactory
```

</p></details>

<details><summary><b>org.pentaho.reporting.engine.classic.extensions.datasources.kettle.KettleDataFactoryTest.testDerive</b> (click to expand)</summary><p>

```
Caught Kettle Exception: Check your configuration
```

</p></details>

<details><summary><b>org.pentaho.reporting.engine.classic.extensions.datasources.kettle.KettleDataFactoryTest.testDesignTimeSupport</b> (click to expand)</summary><p>

```
Caught Kettle Exception: Check your configuration
```

</p></details>

<details><summary><b>org.pentaho.reporting.engine.classic.extensions.datasources.kettle.Prd5056Test.testLoadSave</b> (click to expand)</summary><p>

```
Unable to parse the document: ResourceKey{schema=org.pentaho.reporting.libraries.docbundle.bundleloader.ZipResourceBundleLoader, identifier=content.xml, factoryParameters={org.pentaho.reporting.libraries.resourceloader.FactoryParameterKey{name=repository-loader}=org.pentaho.reporting.libraries.docbundle.bundleloader.ZipResourceBundleLoader@25e8e59, org.pentaho.reporting.libraries.resourceloader.FactoryParameterKey{name=repository}=org.pentaho.reporting.libraries.repository.zipreader.ZipReadRepository@5c72271e}, parent=ResourceKey{schema=org.pentaho.reporting.libraries.resourceloader.loader.raw.RawResourceLoader, identifier=[B@4bc5b2cf, factoryParameters={}, parent=null}}
```

</p></details>

<details><summary><b>org.pentaho.reporting.engine.classic.extensions.datasources.kettle.Prd5056Test.testLoadSaveFalse</b> (click to expand)</summary><p>

```
Unable to parse the document: ResourceKey{schema=org.pentaho.reporting.libraries.docbundle.bundleloader.ZipResourceBundleLoader, identifier=content.xml, factoryParameters={org.pentaho.reporting.libraries.resourceloader.FactoryParameterKey{name=repository-loader}=org.pentaho.reporting.libraries.docbundle.bundleloader.ZipResourceBundleLoader@25e8e59, org.pentaho.reporting.libraries.resourceloader.FactoryParameterKey{name=repository}=org.pentaho.reporting.libraries.repository.zipreader.ZipReadRepository@216ccf2c}, parent=ResourceKey{schema=org.pentaho.reporting.libraries.resourceloader.loader.raw.RawResourceLoader, identifier=[B@1c0c5835, factoryParameters={}, parent=null}}
```

</p></details>

<details><summary><b>org.pentaho.reporting.engine.classic.extensions.datasources.kettle.Prd5062Test.testTransformationNull</b> (click to expand)</summary><p>

```
Unable to parse the document: ResourceKey{schema=org.pentaho.reporting.libraries.docbundle.bundleloader.ZipResourceBundleLoader, identifier=content.xml, factoryParameters={org.pentaho.reporting.libraries.resourceloader.FactoryParameterKey{name=repository-loader}=org.pentaho.reporting.libraries.docbundle.bundleloader.ZipResourceBundleLoader@25e8e59, org.pentaho.reporting.libraries.resourceloader.FactoryParameterKey{name=repository}=org.pentaho.reporting.libraries.repository.zipreader.ZipReadRepository@2d148f03}, parent=ResourceKey{schema=org.pentaho.reporting.libraries.resourceloader.loader.raw.RawResourceLoader, identifier=[B@21cefc6d, factoryParameters={}, parent=null}}
```

</p></details>

<details><summary><b>org.pentaho.reporting.engine.classic.extensions.datasources.kettle.Prd5062Test.testStepNameNull</b> (click to expand)</summary><p>

```
Unable to parse the document: ResourceKey{schema=org.pentaho.reporting.libraries.docbundle.bundleloader.ZipResourceBundleLoader, identifier=content.xml, factoryParameters={org.pentaho.reporting.libraries.resourceloader.FactoryParameterKey{name=repository-loader}=org.pentaho.reporting.libraries.docbundle.bundleloader.ZipResourceBundleLoader@25e8e59, org.pentaho.reporting.libraries.resourceloader.FactoryParameterKey{name=repository}=org.pentaho.reporting.libraries.repository.zipreader.ZipReadRepository@79aac496}, parent=ResourceKey{schema=org.pentaho.reporting.libraries.resourceloader.loader.raw.RawResourceLoader, identifier=[B@6111aa49, factoryParameters={}, parent=null}}
```

</p></details>

<details><summary><b>org.pentaho.reporting.engine.classic.extensions.datasources.kettle.Prd5062Test.testQueryNameEmpty</b> (click to expand)</summary><p>

```
Unable to parse the document: ResourceKey{schema=org.pentaho.reporting.libraries.docbundle.bundleloader.ZipResourceBundleLoader, identifier=content.xml, factoryParameters={org.pentaho.reporting.libraries.resourceloader.FactoryParameterKey{name=repository-loader}=org.pentaho.reporting.libraries.docbundle.bundleloader.ZipResourceBundleLoader@25e8e59, org.pentaho.reporting.libraries.resourceloader.FactoryParameterKey{name=repository}=org.pentaho.reporting.libraries.repository.zipreader.ZipReadRepository@283b462d}, parent=ResourceKey{schema=org.pentaho.reporting.libraries.resourceloader.loader.raw.RawResourceLoader, identifier=[B@18a7c1f6, factoryParameters={}, parent=null}}
```

</p></details>

<details><summary><b>org.pentaho.reporting.engine.classic.extensions.datasources.kettle.Prd5062Test.testTransformationNameEmpty</b> (click to expand)</summary><p>

```
Unable to parse the document: ResourceKey{schema=org.pentaho.reporting.libraries.docbundle.bundleloader.ZipResourceBundleLoader, identifier=content.xml, factoryParameters={org.pentaho.reporting.libraries.resourceloader.FactoryParameterKey{name=repository-loader}=org.pentaho.reporting.libraries.docbundle.bundleloader.ZipResourceBundleLoader@25e8e59, org.pentaho.reporting.libraries.resourceloader.FactoryParameterKey{name=repository}=org.pentaho.reporting.libraries.repository.zipreader.ZipReadRepository@2d148f03}, parent=ResourceKey{schema=org.pentaho.reporting.libraries.resourceloader.loader.raw.RawResourceLoader, identifier=[B@21cefc6d, factoryParameters={}, parent=null}}
```

</p></details>

<details><summary><b>org.pentaho.reporting.engine.classic.extensions.datasources.kettle.Prd5062Test.testStepNameEmpty</b> (click to expand)</summary><p>

```
Unable to parse the document: ResourceKey{schema=org.pentaho.reporting.libraries.docbundle.bundleloader.ZipResourceBundleLoader, identifier=content.xml, factoryParameters={org.pentaho.reporting.libraries.resourceloader.FactoryParameterKey{name=repository-loader}=org.pentaho.reporting.libraries.docbundle.bundleloader.ZipResourceBundleLoader@25e8e59, org.pentaho.reporting.libraries.resourceloader.FactoryParameterKey{name=repository}=org.pentaho.reporting.libraries.repository.zipreader.ZipReadRepository@ea3a636}, parent=ResourceKey{schema=org.pentaho.reporting.libraries.resourceloader.loader.raw.RawResourceLoader, identifier=[B@661a26a8, factoryParameters={}, parent=null}}
```

</p></details>

<details><summary><b>org.pentaho.reporting.engine.classic.extensions.datasources.mondrian.EmptyCellsetTest.testEmptyResult</b> (click to expand)</summary><p>

```
results should be empty, rowcount should be 0. expected:<0> but was:<1>
```

</p></details>

  
Tests run: **2646**, Failures: **25**, Skipped: **15** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/reporting/job/PR-1639/1/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :x: Build failed in 21m 29s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -Djs.no.sandbox
```
---
### :no_entry: Failed Tests

#### :cloud_with_lightning_and_rain: 22 test(s) failed:

<details><summary><b>org.pentaho.reporting.engine.classic.extensions.datasources.kettle.BigDataKettleFactoryTest.testParameter</b> (click to expand)</summary><p>

```
Unable to locate metadata for data-factory type org.pentaho.reporting.engine.classic.extensions.datasources.kettle.KettleDataFactory
```

</p></details>

<details><summary><b>org.pentaho.reporting.engine.classic.extensions.datasources.kettle.BigDataKettleFactoryTest.testQuery</b> (click to expand)</summary><p>

```
Caught Kettle Exception: Check your configuration
```

</p></details>

<details><summary><b>org.pentaho.reporting.engine.classic.extensions.datasources.kettle.BigDataKettleFactoryTest.testSaveAndLoad</b> (click to expand)</summary><p>

```
Unable to parse the document: ResourceKey{schema=org.pentaho.reporting.libraries.docbundle.bundleloader.ZipResourceBundleLoader, identifier=content.xml, factoryParameters={org.pentaho.reporting.libraries.resourceloader.FactoryParameterKey{name=repository-loader}=org.pentaho.reporting.libraries.docbundle.bundleloader.ZipResourceBundleLoader@25e8e59, org.pentaho.reporting.libraries.resourceloader.FactoryParameterKey{name=repository}=org.pentaho.reporting.libraries.repository.zipreader.ZipReadRepository@6bd679b4}, parent=ResourceKey{schema=org.pentaho.reporting.libraries.resourceloader.loader.raw.RawResourceLoader, identifier=[B@1f3b3c33, factoryParameters={}, parent=null}}
```

</p></details>

<details><summary><b>org.pentaho.reporting.engine.classic.extensions.datasources.kettle.BigDataKettleFactoryTest.testSerialize</b> (click to expand)</summary><p>

```
Caught Kettle Exception: Check your configuration
```

</p></details>

<details><summary><b>org.pentaho.reporting.engine.classic.extensions.datasources.kettle.BigDataKettleFactoryTest.testMetaData</b> (click to expand)</summary><p>

```
Unable to locate metadata for data-factory type org.pentaho.reporting.engine.classic.extensions.datasources.kettle.KettleDataFactory
```

</p></details>

<details><summary><b>org.pentaho.reporting.engine.classic.extensions.datasources.kettle.BigDataKettleFactoryTest.testDerive</b> (click to expand)</summary><p>

```
Caught Kettle Exception: Check your configuration
```

</p></details>

<details><summary><b>org.pentaho.reporting.engine.classic.extensions.datasources.kettle.BigDataKettleFactoryTest.testSaveAndLoadForSubReports</b> (click to expand)</summary><p>

```
Unable to parse the document: ResourceKey{schema=org.pentaho.reporting.libraries.docbundle.bundleloader.ZipResourceBundleLoader, identifier=content.xml, factoryParameters={org.pentaho.reporting.libraries.resourceloader.FactoryParameterKey{name=repository-loader}=org.pentaho.reporting.libraries.docbundle.bundleloader.ZipResourceBundleLoader@25e8e59, org.pentaho.reporting.libraries.resourceloader.FactoryParameterKey{name=repository}=org.pentaho.reporting.libraries.repository.zipreader.ZipReadRepository@5d8a0b6}, parent=ResourceKey{schema=org.pentaho.reporting.libraries.resourceloader.loader.raw.RawResourceLoader, identifier=[B@6b8564e5, factoryParameters={}, parent=null}}
```

</p></details>

<details><summary><b>org.pentaho.reporting.engine.classic.extensions.datasources.kettle.KettleDataFactoryTest.testParameter</b> (click to expand)</summary><p>

```
Unable to locate metadata for data-factory type org.pentaho.reporting.engine.classic.extensions.datasources.kettle.KettleDataFactory
```

</p></details>

<details><summary><b>org.pentaho.reporting.engine.classic.extensions.datasources.kettle.KettleDataFactoryTest.testQuery</b> (click to expand)</summary><p>

```
Caught Kettle Exception: Check your configuration
```

</p></details>

<details><summary><b>org.pentaho.reporting.engine.classic.extensions.datasources.kettle.KettleDataFactoryTest.testSaveAndLoad</b> (click to expand)</summary><p>

```
Unable to parse the document: ResourceKey{schema=org.pentaho.reporting.libraries.docbundle.bundleloader.ZipResourceBundleLoader, identifier=content.xml, factoryParameters={org.pentaho.reporting.libraries.resourceloader.FactoryParameterKey{name=repository-loader}=org.pentaho.reporting.libraries.docbundle.bundleloader.ZipResourceBundleLoader@25e8e59, org.pentaho.reporting.libraries.resourceloader.FactoryParameterKey{name=repository}=org.pentaho.reporting.libraries.repository.zipreader.ZipReadRepository@68f1b06b}, parent=ResourceKey{schema=org.pentaho.reporting.libraries.resourceloader.loader.raw.RawResourceLoader, identifier=[B@56c802bc, factoryParameters={}, parent=null}}
```

</p></details>

<details><summary><b>org.pentaho.reporting.engine.classic.extensions.datasources.kettle.KettleDataFactoryTest.testSerialize</b> (click to expand)</summary><p>

```
Caught Kettle Exception: Check your configuration
```

</p></details>

<details><summary><b>org.pentaho.reporting.engine.classic.extensions.datasources.kettle.KettleDataFactoryTest.testMetaData</b> (click to expand)</summary><p>

```
Unable to locate metadata for data-factory type org.pentaho.reporting.engine.classic.extensions.datasources.kettle.KettleDataFactory
```

</p></details>

<details><summary><b>org.pentaho.reporting.engine.classic.extensions.datasources.kettle.KettleDataFactoryTest.testDerive</b> (click to expand)</summary><p>

```
Caught Kettle Exception: Check your configuration
```

</p></details>

<details><summary><b>org.pentaho.reporting.engine.classic.extensions.datasources.kettle.KettleDataFactoryTest.testDesignTimeSupport</b> (click to expand)</summary><p>

```
Caught Kettle Exception: Check your configuration
```

</p></details>

<details><summary><b>org.pentaho.reporting.engine.classic.extensions.datasources.kettle.Prd5056Test.testLoadSave</b> (click to expand)</summary><p>

```
Unable to parse the document: ResourceKey{schema=org.pentaho.reporting.libraries.docbundle.bundleloader.ZipResourceBundleLoader, identifier=content.xml, factoryParameters={org.pentaho.reporting.libraries.resourceloader.FactoryParameterKey{name=repository-loader}=org.pentaho.reporting.libraries.docbundle.bundleloader.ZipResourceBundleLoader@25e8e59, org.pentaho.reporting.libraries.resourceloader.FactoryParameterKey{name=repository}=org.pentaho.reporting.libraries.repository.zipreader.ZipReadRepository@5c72271e}, parent=ResourceKey{schema=org.pentaho.reporting.libraries.resourceloader.loader.raw.RawResourceLoader, identifier=[B@4bc5b2cf, factoryParameters={}, parent=null}}
```

</p></details>

<details><summary><b>org.pentaho.reporting.engine.classic.extensions.datasources.kettle.Prd5056Test.testLoadSaveFalse</b> (click to expand)</summary><p>

```
Unable to parse the document: ResourceKey{schema=org.pentaho.reporting.libraries.docbundle.bundleloader.ZipResourceBundleLoader, identifier=content.xml, factoryParameters={org.pentaho.reporting.libraries.resourceloader.FactoryParameterKey{name=repository-loader}=org.pentaho.reporting.libraries.docbundle.bundleloader.ZipResourceBundleLoader@25e8e59, org.pentaho.reporting.libraries.resourceloader.FactoryParameterKey{name=repository}=org.pentaho.reporting.libraries.repository.zipreader.ZipReadRepository@216ccf2c}, parent=ResourceKey{schema=org.pentaho.reporting.libraries.resourceloader.loader.raw.RawResourceLoader, identifier=[B@1c0c5835, factoryParameters={}, parent=null}}
```

</p></details>

<details><summary><b>org.pentaho.reporting.engine.classic.extensions.datasources.kettle.Prd5062Test.testTransformationNull</b> (click to expand)</summary><p>

```
Unable to parse the document: ResourceKey{schema=org.pentaho.reporting.libraries.docbundle.bundleloader.ZipResourceBundleLoader, identifier=content.xml, factoryParameters={org.pentaho.reporting.libraries.resourceloader.FactoryParameterKey{name=repository-loader}=org.pentaho.reporting.libraries.docbundle.bundleloader.ZipResourceBundleLoader@25e8e59, org.pentaho.reporting.libraries.resourceloader.FactoryParameterKey{name=repository}=org.pentaho.reporting.libraries.repository.zipreader.ZipReadRepository@2d148f03}, parent=ResourceKey{schema=org.pentaho.reporting.libraries.resourceloader.loader.raw.RawResourceLoader, identifier=[B@21cefc6d, factoryParameters={}, parent=null}}
```

</p></details>

<details><summary><b>org.pentaho.reporting.engine.classic.extensions.datasources.kettle.Prd5062Test.testStepNameNull</b> (click to expand)</summary><p>

```
Unable to parse the document: ResourceKey{schema=org.pentaho.reporting.libraries.docbundle.bundleloader.ZipResourceBundleLoader, identifier=content.xml, factoryParameters={org.pentaho.reporting.libraries.resourceloader.FactoryParameterKey{name=repository-loader}=org.pentaho.reporting.libraries.docbundle.bundleloader.ZipResourceBundleLoader@25e8e59, org.pentaho.reporting.libraries.resourceloader.FactoryParameterKey{name=repository}=org.pentaho.reporting.libraries.repository.zipreader.ZipReadRepository@79aac496}, parent=ResourceKey{schema=org.pentaho.reporting.libraries.resourceloader.loader.raw.RawResourceLoader, identifier=[B@6111aa49, factoryParameters={}, parent=null}}
```

</p></details>

<details><summary><b>org.pentaho.reporting.engine.classic.extensions.datasources.kettle.Prd5062Test.testQueryNameEmpty</b> (click to expand)</summary><p>

```
Unable to parse the document: ResourceKey{schema=org.pentaho.reporting.libraries.docbundle.bundleloader.ZipResourceBundleLoader, identifier=content.xml, factoryParameters={org.pentaho.reporting.libraries.resourceloader.FactoryParameterKey{name=repository-loader}=org.pentaho.reporting.libraries.docbundle.bundleloader.ZipResourceBundleLoader@25e8e59, org.pentaho.reporting.libraries.resourceloader.FactoryParameterKey{name=repository}=org.pentaho.reporting.libraries.repository.zipreader.ZipReadRepository@283b462d}, parent=ResourceKey{schema=org.pentaho.reporting.libraries.resourceloader.loader.raw.RawResourceLoader, identifier=[B@18a7c1f6, factoryParameters={}, parent=null}}
```

</p></details>

<details><summary><b>org.pentaho.reporting.engine.classic.extensions.datasources.kettle.Prd5062Test.testTransformationNameEmpty</b> (click to expand)</summary><p>

```
Unable to parse the document: ResourceKey{schema=org.pentaho.reporting.libraries.docbundle.bundleloader.ZipResourceBundleLoader, identifier=content.xml, factoryParameters={org.pentaho.reporting.libraries.resourceloader.FactoryParameterKey{name=repository-loader}=org.pentaho.reporting.libraries.docbundle.bundleloader.ZipResourceBundleLoader@25e8e59, org.pentaho.reporting.libraries.resourceloader.FactoryParameterKey{name=repository}=org.pentaho.reporting.libraries.repository.zipreader.ZipReadRepository@2d148f03}, parent=ResourceKey{schema=org.pentaho.reporting.libraries.resourceloader.loader.raw.RawResourceLoader, identifier=[B@21cefc6d, factoryParameters={}, parent=null}}
```

</p></details>

<details><summary><b>org.pentaho.reporting.engine.classic.extensions.datasources.kettle.Prd5062Test.testStepNameEmpty</b> (click to expand)</summary><p>

```
Unable to parse the document: ResourceKey{schema=org.pentaho.reporting.libraries.docbundle.bundleloader.ZipResourceBundleLoader, identifier=content.xml, factoryParameters={org.pentaho.reporting.libraries.resourceloader.FactoryParameterKey{name=repository-loader}=org.pentaho.reporting.libraries.docbundle.bundleloader.ZipResourceBundleLoader@25e8e59, org.pentaho.reporting.libraries.resourceloader.FactoryParameterKey{name=repository}=org.pentaho.reporting.libraries.repository.zipreader.ZipReadRepository@ea3a636}, parent=ResourceKey{schema=org.pentaho.reporting.libraries.resourceloader.loader.raw.RawResourceLoader, identifier=[B@661a26a8, factoryParameters={}, parent=null}}
```

</p></details>

<details><summary><b>org.pentaho.reporting.engine.classic.extensions.datasources.mondrian.EmptyCellsetTest.testEmptyResult</b> (click to expand)</summary><p>

```
results should be empty, rowcount should be 0. expected:<0> but was:<1>
```

</p></details>

  
Tests run: **2507**, Failures: **22**, Skipped: **14** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/reporting/job/PR-1639/2/testReport)

 ---
### Errors:
<details><summary><b>Filtered log</b> (click to expand)</summary>
<p>

```
script returned exit code 1
```
 
</p>
</details>
 
 

:information_source:  _This is an automatic message_

File: pom.xml [hunk 1]
```diff
@@ -160,6 +160,7 @@
     <pentaho-metaverse.version>10.2.0.0-SNAPSHOT</pentaho-metaverse.version>
     <encryption-support.version>10.2.0.0-SNAPSHOT</encryption-support.version>
     <jakarta-annotation-api.version>1.3.5</jakarta-annotation-api.version>
+    <safeyaml.version>1.34.1</safeyaml.version>
```
[the remark this answers is not in the document]
Comment (andreramos89) [reply pom.xml:163]: In the build pipeline it seemed to fail when the version was not present. 
It probably is a recursive dependency issue in the build pipeline, but without it the build would not be successful.

**Review by smmribeiro — APPROVED** [review]
Looks good.

**Pull request merged**